pkg/utils/nodeclass/nodeclass.go

```diff
@@ -97,7 +97,7 @@ func NewSecurityGroupSelectorTerms(securityGroupSelector map[string]string) (ter
 	for k, v := range securityGroupSelector {
 		switch k {
 		case "aws-ids", "aws::ids":
-			ids = strings.Split(strings.Trim(v, " "), ",")
+			ids = functional.SplitCommaSeparatedString(v)
```

Comment:
Do we need to build this logic into a bespoke function or is it sufficient to do the following that way all of the logic is there

```go
ids = lo.Map(strings.Split(v, ","), func(s string, _ int) string { return strings.Trim(s, " ") }
```

Reply:
I could also see the helper function live inside of this file. I'm a bit unconvinced that we need a project-wide helper for this specific logic